code/src/components/HappyPage.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react';
+
+import HappyList from 'components/HappyList';
+import HappyForm from 'components/HappyForm'
+
+const HappyPage = () => {
+  const [thoughtList, setThoughtList] = useState([]);
+  const [newThought, setNewThought] = useState('');
+
+  const getHappyThoughts = () => {
+    fetch('https://happy-thoughts-technigo.herokuapp.com/thoughts')
```

Comment:
You could also go a step further declaring a variable with the url and then calling the variable inside the fetch. I.e: 

```
const APIurl = 'https://happy-thoughts-technigo.herokuapp.com/thoughts'

  const getHappyThoughts = () => {
    fetch(APIurl)
      .then((data) => data.json())
      .then((transformedData) => setThoughtList(transformedData))
      .catch((error) => console.error(error))
      .finally(() => console.log('no errors'))
  }
```